src/content/service.js

```diff
@@ -185,20 +186,23 @@ module.exports = async ({ botfile, projectLocation, logger }) => {
         .then()
     } else {
       const randomId = getNewItemId(category)
-      await knex('content_items')
-        .insert({
-          ...body,
-          createdBy: 'admin',
-          createdOn: helpers(knex).date.now(),
-          id: randomId,
-          categoryId
-        })
-        .then()
+      await knex('content_items').insert({
+        ...body,
+        createdBy: 'admin',
+        createdOn: helpers(knex).date.now(),
+        id: randomId,
+        categoryId
+      })
     }
 
     return dumpDataToFile(categoryId)
   }
 
+  const categoryItemsCount = async () => {
+    const [result] = await knex('content_items').count('id')
+    return result['count("id")']
```

Comment:
If you do the `as` thing (an be seen in my code in this file) you won't need this string key syntax